contracts/RelayerHub.sol

```diff
@@ -71,7 +72,7 @@ contract RelayerHub is IRelayerHub, System, IParamSubscriber{
     Relayer memory r = relayers[msg.sender];
     delete relayers[msg.sender];
     payable(msg.sender).transfer(r.deposit - r.dues); //@dev:safe(no DoS since msg.sender)
-    payable(SYSTEM_REWARD_ADDR).transfer(r.dues);
```

Comment:
note that SystemReward may no longer receive eth via send/transfer due to additional logic added to its receive()

Reply:
Been through this a few times and it looks safe. The more we change the more I realize that we should create a 'best practices' document in our internal eng notion to track all these specific implementation standards. Our (reasonable) changes here and forcing future developers to adhere to the new standards - a standards doc for internal alignment or onboarding will probably be required shortly